src/scheduler/TimeGrid.js

```diff
@@ -319,6 +319,9 @@ export default class TimeGrid extends Component {
         content.scrollLeft =
           (content.scrollWidth - gutterWidth) * this._scrollRatio
         // Only do this once
+        setTimeout(() => {
+          this._scrollRatio = null
+        }, 500)
```

Comment:
Looks like, this value is used somewhere else and should not be set to null here. This feel like a hack and its hard to say where it'll fail now. Correct thing to do will be to identify where this value is used after this method and try to fix it properly. Calendar is very critical and adding a timeout does not feel correct. I can also take a look at it tonite, if you can point me to exact scenario that is broken. @victor-guillen-housecall